package.json

```diff
@@ -12,8 +12,7 @@
     "md:lint": "npx markdownlint-cli '*.md' 'docs/*.md'",
     "md:spell": "npx markdown-spellcheck -a -n -x --en-us '*.md' 'docs/*.md'",
     "test": "echo \"Error: testing disabled\" && exit 0",
-    "package": "web-ext build",
-    "build": "web-ext build",
+    "build": "web-ext build --filename secure-proxy.xpi",
```

Comment:
I haven’t checked the docs/code, but not sure if you want to put this in the webext config file with the other settings. 🤷

Reply:
it seems it's not possible to set the filename in the webext config file.